### DISCUSSION TITLE
yiwen-blog-website/blog/2024_0513

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#263] [Announcements]
# yiwen-blog-website/blog/2024_0513

今天突发奇想，因为很多时候个人项目进展极其缓慢，我一直在思考原因在什么地方。经过一段时间的思考，我发觉是效率，因为没有使用现代化的企业工具导致的效率落后，那么我在思考，如果使用效率工具，是否会对项目进展有一些提升。于是我决定在接下来的项目开发中使用飞书来提效。

https://hao-yiwen.github.io/yiwen-blog-website/blog/2024_0513

**Comment by Hao-yiwen** [comment]
notion效果完全优于飞书。所以用notion就好。
